tools/pipelines/templates/build-npm-package.yml

```diff
@@ -620,3 +628,38 @@ stages:
         namespace: ${{ parameters.namespace }}
         tagName: ${{ parameters.tagName }}
         publishNonScopedPackages: ${{ variables.publishNonScopedPackages }}
+
+  # Capture per-pipeline stage results
+  - stage: runAfterAll
+    displayName: Retrieve and upload client stage stats to Kusto
+    condition: and(succeededOrFailed(), ne(variables['Build.Reason'], 'PullRequest'), eq(${{parameters.telemetry}}, true))
+    dependsOn: 
+      - build
+
+    jobs:
+      - template: include-test-perf-benchmarks.yml
```

Comment:
This looks... weird. Should that shared part of it be refactored or renamed?

Reply:
Would something like `include-telemetry-setup.yml` make more sense? I would be less confident in refactoring the template than renaming.

Reply:
Yeah, that seems fine.

Reply:
Yeah, I noticed the same thing before but hadn't gotten around to renaming the template. I like `include-telemetry-setup.yml`. For reference, I'll create an ADO task for refactor so the job is not defined within the template (right now it creates a job named "Run tests" which might not apply in all places where the template is used). For now the rename is good.